docs/src/views/FocusZone.tsx

```diff
@@ -0,0 +1,365 @@
+        {link('Roving tabindex', 'https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex')}
+      </li>
+    </ul>
+    <p>
+      Stardust leverages Focus Zone component which is based on the{' '}
+      {link(
+        'Focus Zone from Office UI Fabric',
+        'https://developer.microsoft.com/en-us/fabric#/components/focuszone',
+      )}
+      . This component allows to wrap any component/element and adds arrow key navigation
+      functionality.
+    </p>
+    <Header as="h2">Usage</Header>
+    <div>
+      In Stardust, Focus Zone is assigned to components through accessibility behaviors
+      <CodeSnippet
+        label="menuBehavior.ts"
+        value={`
+        const menuBehavior: Accessibility = (props: any) => ({
+          //...
+          focusZone: {
+            mode: FocusZoneMode.Embed,
+            props: {
+              //...
+            },
+          },
+        })
+        `}
+      />
+      and then accessibility behavior is set for component
+      <CodeSnippet
+        label="NavigableMenu.jsx"
+        value={`
+        const NavigableMenu = () => (
+          <Menu accessibility={menuBehavior} />
+        )`}
+      />
+      Read more about <b>accessibility behaviors {link('here', '/accessibility-behaviors')}</b>.
+    </div>
+    <p>
+      For setting Focus Zone properly in accessibility behavior, it is needed to specify FocusZone's{' '}
+      <b>mode</b> and <b>props</b>.
+    </p>
+    <Header as="h3">Mode</Header>
+    <p>Type: {code('FocusZoneMode')}, with 2 main options:</p>
+    <ul>
+      <li>
+        <b>Embed</b> - Focus Zone is embeded into component's container, thus all FocusZone's
+        attributes/events listeners are applied to component's container.
+        <CodeSnippet
+          label="html"
+          value={`
+          <ul role="menu" class="ui-menu">---> FocusZone's attributes/events listeners applied here 
+            <li class="ui-menu__item ..." role="presentation">
+            </li>
+            <li class="ui-menu__item ..." role="presentation">
+            </li>
+          </ul>
+          `}
+        />
+      </li>
+      <li>
+        <b>Wrap</b> - Focus Zone wraps component to it's own container and all attributes/events
+        listeners applied there.
+        <CodeSnippet
+          label="html"
+          value={`
+            <div>---> FocusZone's attributes/events listeners applied here            
+              <ul role="menu" class="ui-menu">
+                <li class="ui-menu__item ..." role="presentation">
+                </li>
+                <li class="ui-menu__item ..." role="presentation">
+                </li>
+              </ul>
+            </div>
+          `}
+        />
+      </li>
+    </ul>
+    <Header as="h3">Props</Header>
+    <p>To FocusZone next props can be applied:</p>
+    <ul>
+      <li>
+        <b>as</b> - Element type the root element will use. Default is "div".
```

Comment:
not sure if we want to document `as`. maybe mark it as internal? the wrap/embed logic is overriding it as far as I remember

Reply:
if you specify "as" for wrap logic, it will override "div". But true, that for Embed it won't work as it respects component's ElementType. Will specify that